programs/rodeos/cloner_plugin.cpp

```diff
@@ -258,7 +260,8 @@ void cloner_plugin::plugin_initialize(const variables_map& options) {
       if (options.count("telemetry-url")) {
          fc::zipkin_config::init( options["telemetry-url"].as<std::string>(),
                                   options["telemetry-service-name"].as<std::string>(),
-                                  options["telemetry-timeout-us"].as<uint32_t>() );
+                                  options["telemetry-timeout-us"].as<uint32_t>(),
+                                  options["telemetry-retry-interval-us"].as<uint32_t>() );
       }
```

Comment:
Add `fc::zipkin_config::handle_sighup();` to `cloner_plugin::handle_sighup()`

Reply:
Added.